src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/model/TaskProgress.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Task
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.TaskStatus
+
+@Schema(description = "Response object which describes an assessment")
```

Comment:
I guess this needs updating from "an assessment"

Reply:
thanks updated that